roles/common/defaults/main.yml

```diff
@@ -87,6 +87,9 @@ diskgroup_compatible_rdbms: "{# assign to variable c compatible parameter and se
 
 run_cluvfy: false
 
+# section for DR
+standby_suffix: s
```

Comment:
I like to add underscores like "_s" or "_sby" to make it clear that this is a suffix